src/main/java/com/stories/mapper/StoryMapping.java

```diff
@@ -0,0 +1,18 @@
+package com.stories.mapper;
+
+
+import com.stories.domain.StoryDomain;
+import com.stories.model.StoryModel;
+
+import ma.glasnost.orika.MapperFactory;
+import net.rakugakibox.spring.boot.orika.OrikaMapperFactoryConfigurer;
+
+//implements OrikaMapperFactoryConfigurer
+public class StoryMapping implements OrikaMapperFactoryConfigurer {
+
+	@Override
+	public void configure(MapperFactory orikaMapperFactory){
+		orikaMapperFactory.classMap(StoryDomain.class, StoryModel.class).byDefault().register();
+	}
+
+}
```

Comment:
Where are you using this StoryMapping class in your application?

Reply:
on the controller, but this not working.

Reply:
Please delete it if you're not using it.